internal/adapter/etcd.go

```diff
@@ -217,8 +217,9 @@ func (a *adapter) handleDeleteEvent(ctx context.Context, ev *Event) {
 }
 
 func (a *adapter) showVersion(w http.ResponseWriter, _ *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusOK)
-	_, err := w.Write([]byte(`{"etcdserver":"3.5.0-pre","etcdcluster":"3.5.0"}`))
+	_, err := w.Write([]byte(`{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}`))
```

Comment:
Is this to use a fixed version that will need to be modified each time the ETCD is updated in the future?

Reply:
No need, we just need to use 3.5.0 all the time.